Samples/SparsePackages/WPF+SparsePkg.sln

```diff
@@ -15,6 +15,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "PhotoStoreContextMenu", "PhotoStoreContextMenu\PhotoStoreContextMenu.vcxproj", "{16746B0C-5840-4ED4-AEDF-1ABB617827FD}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "SparseCppApp", "SparseCppApp\SparseCppApp.vcxproj", "{1A597D15-7EF3-47F2-B89D-D180CBAA0F32}"

```

Comment:
We need to refactor the folder structure here:

PackageWithExternalLocation (rename SparsePackages folder to this new name)
|
|----> cs
|--------> Existing "SparsePackages" folder content with the WPF solution
|----> cppwinrt
--------> New cppwinrt sample you created in its own solution

Reply:
https://github.com/microsoft/AppModelSamples/pull/44